dist/js/medium-editor.js

```diff
@@ -4267,11 +4408,11 @@ function MediumEditor(elements, options) {
         },
 
         cleanPaste: function (text) {
-            this.pasteHandler.cleanPaste(text);
+            this.getExtensionByName('paste').cleanPaste(text);
         },
 
         pasteHTML: function (html, options) {
-            this.pasteHandler.pasteHTML(html, options);
+            this.getExtensionByName('paste').pasteHTML(html, options);
```

Comment:
Yeah, because I wanted to think through those parts more, I purposefully made it so you can't turn off the pastehandler.  I want to consider what you're suggesting vs keeping everything in the extensions array in another iteration after these changes get in.

For now, this is a safe workaround without making a decision one way or another.